var/spack/repos/builtin/packages/sundials/package.py

```diff
@@ -19,6 +19,7 @@ class Sundials(CMakePackage):
     # ==========================================================================
     # Versions
     # ==========================================================================
+    version('5.0.0-dev.1', sha256='8c8e4fcb86497f8625b5dc38c9b6257c6e3097c472564436e2900de1d3d02206')
     version('4.1.0', sha256='280de1c27b2360170a6f46cb3799b2aee9dff3bddbafc8b08c291a47ab258aa5')
```

Comment:
You should make the previous release the default since this dev release isn't likely stable.
```suggestion
    version('4.1.0', sha256='280de1c27b2360170a6f46cb3799b2aee9dff3bddbafc8b08c291a47ab258aa5', preferred=True)
```